tinkerhost.net.th-customdomain.json

```diff
@@ -0,0 +1,28 @@
+{
+  "providerId": "tinkerhost.net",
+  "providerName": "TinkerHost",
+  "serviceId": "th-customDomain",
+  "serviceName": "TinkerHost Hosting",
+  "version": 1,
+  "syncPubKeyDomain": "domainConnect.tinkerhost.net",

```

Comment:
it is not an errror, but out of purity please just use low caps on host names

Reply:
Not a problem, updated to resolve